omero/sysadmins/index.txt

```diff
@@ -59,6 +59,19 @@ performance and security.
     omero-home-prefix
     config
 
+*****************************************
+Optimizing and Publishing Data Repository
+*****************************************
+
+This chapter explains how to customize your OMERO data repository.
```

Comment:
…customize the appearance and functionality of OMERO clients to host images for groups or public viewing.